documentation/src/docs/deployment/docker-swarm.md

```diff
@@ -26,7 +26,7 @@ docker stack up --compose-file zookeeper.yml pravega
 ```
 
 This runs a single node HDFS container and single node ZooKeeper inside the `pravega_default` overlay network, and adds
-them to the `pravega`stack.
+them to the pravegastack.
```

Comment:
This may be written as "Pravega `stack`" or just "Pravega stack"? (from Docker `stack up/down` command)

Reply:
I guess this should be "`pravega` stack", where:
- "`pravega`" is the name of the stack.
- "stack" is just a noun.

Reply:
@adrian leaving it as `Pravega` stack

Reply:
`pravega` stack